rad_eap_test

```diff
@@ -793,6 +805,20 @@ function check_settings()
     SSID="eduroam";
   fi
 
+  if [[ -n "$TLS_PROTOCOLS" ]]
+  then
+    TLS_PROTOCOLS=",$TLS_PROTOCOLS," # for easier matching
+
+    # For each TLS protocol, if included in the list, exit value will be 0 and the protocol will be enabled.
+    [[ $TLS_PROTOCOLS =~ ,TLSv1, ]] ; tls10_disabled=$?
+    [[ $TLS_PROTOCOLS =~ ,TLSv1\.1, ]] ; tls11_disabled=$?
+    [[ $TLS_PROTOCOLS =~ ,TLSv1\.2, ]] ; tls12_disabled=$?
+    [[ $TLS_PROTOCOLS =~ ,TLSv1\.3, ]] ; tls13_disabled=$?
+
+    PHASE1="$PHASE1 tls_disable_tlsv1_0=$tls10_disabled tls_disable_tlsv1_1=$tls11_disabled tls_disable_tlsv1_2=$tls12_disabled tls_disable_tlsv1_3=$tls13_disabled"
```

Comment:
Out of interest, shouldn't PHASE1 be handled separately from TLS_PROTOCOLS? :-)

Reply:
Sorry, it's my bad - I confused phase1 with the phase2 stuff (i.e. the inner tunnel) which is separate from phase1. But phase1 options include TLS options, so it makes sense then. Apologies! @michalvasko, can someone from CESNET approve and merge this pull request, please? And also #19, which is related to this too? Pretty please? We appreciate you folks at CESNET very much :-)

Reply:
@medved-cz, is that you? The maintainer that is?